giles-eco-tardis/src/main/java/edu/asu/diging/tardis/core/service/impl/InnogenScriptRunner.java

```
@@ -0,0 +1,60 @@
+package edu.asu.diging.tardis.core.service.impl;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.nio.file.Path;
+
+import edu.asu.diging.gilesecosystem.septemberutil.properties.MessageType;
+import edu.asu.diging.gilesecosystem.septemberutil.service.ISystemMessageHandler;
+import edu.asu.diging.gilesecosystem.util.properties.IPropertiesManager;
+import edu.asu.diging.tardis.config.Properties;
+import edu.asu.diging.tardis.core.service.IInnogenScriptRunner;
+
+@Service
+public class InnogenScriptRunner implements IInnogenScriptRunner{
+    
+    @Autowired
+    protected ISystemMessageHandler messageHandler;
+    
+    @Autowired
+    protected IPropertiesManager propertiesManager;
+    
+    @Override
+    public void runInnogenScript(String imagePath, int pareNr) {
```

this method needs to let the caller know if there was an error or if this was successful. I would probably throw an exception if the docker command throws one.